guide/content/_index.md

```diff
@@ -3,14 +3,11 @@ title : "Welcome to the Gameface Starter Guide!"
 description: ""
 lead: "
 
-Gameface is for developers who’d like to utilize modern HTML5 and JavaScript to create a game user interface. Gameface shortens development time by allowing you to use modern and widely available technologies.
-<br>
-<br>
-These are a series of tutorials focused on building a UI and using it in your game engine.
 
 "
 date: 2020-10-06T08:47:36+00:00
 lastmod: 2020-10-06T08:47:36+00:00
 draft: false
 images: []
 ---
+Gameface is for developers who’d like to utilize modern HTML5 and JavaScript to create a game user interface. Gameface shortens development time by allowing you to use modern and widely available technologies. <br> <br> These are a series of tutorials focused on building a UI and using it in your game engine.
```

Comment:
Do we need <br><br> here? Can't we simply add an empty line(s)?

Reply:
Fixed.